### DISCUSSION TITLE
2023/08/the-price/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#89] [General]
# 2023/08/the-price/

https://xiangyun.rbind.io/2023/08/the-price/

**Comment by XiangyunHuang** [comment]
我觉得 AI 不可能在绘画、电影、昆曲、话剧等艺术中产生革命性影响。绘画、昆曲、话剧属于一种现场艺术，强调当下的感受，特别是昆曲和绘画，弄 AI 这种虚的传递不出强烈的感官刺激。AI 对电影、动漫可能会有些助力，但不是主导的，电影、动漫主要还是导演、剧本，AI 在后期特效制作方面会有帮助，在通俗片中的使用会比文艺片更广。
